### DISCUSSION TITLE
ProductType?

### DISCUSSION THREAD

**Discussion by asucrews** [zwave-js/zwave-js#6247] [Q&A] [answered]
i was reading though the config.json for each device and I was wouderning what ProductType is or better what the number means?  i could find a list what manufacturer id are, but I can not seem to find one for PtodcutType.

**Comment by kpine** [comment, marked as answer]
They are uniquely defined by each manufacturer. Manufacturers may or may not use them to categorize their device models in certain ways.

![image](https://github.com/zwave-js/node-zwave-js/assets/791794/e3d07749-d890-499c-aa8c-7f20f4e3171f)

https://github.com/zwave-js/specs/blob/master/Z-Wave%20Specification%20AWG%20V3.0.pdf

**Comment by asucrews** [comment]
Cool Thanks!
